proarc-webapp/src/main/java/cz/cas/lib/proarc/webapp/client/widget/mods/NdkPeriodicalIssueForm.java

```diff
@@ -168,7 +168,7 @@ private Field name() {
                     .addMapValue("primary", "primary")
                 .createField()) // usage
                 // namePart, namePartDefinition extends stringPlusLanguage
-                .addField(new FieldBuilder("namePart").setTitle("Name Parts - MA").setMaxOccurrences(5)
+                .addField(new FieldBuilder("namePart").setTitle("Name Parts - M").setMaxOccurrences(5).setRequired(true)
```

Comment:
Má smysl nastavovat `.setRequired(true)` pro vložený formulář? Testoval jsi to? Zběžně jsem prošel zdrojáky generátoru formulářů a tohle jsem nenašel. Vidím, že to nastavuješ na několika místech.

Reply:
Nested form se generuje v https://github.com/proarc/proarc/blob/master/proarc-webapp/src/main/java/cz/cas/lib/proarc/webapp/client/widget/form/FormGenerator.java#L153

Field required není na nested formuláře aplikovatelný a pomocí smartgwt není ani možné na celý formulář required nastavit. Proto je to zbytečné a osobně bych připsal vyhození výjimky, pokud se o to někdo pokusí, např. takto
```
    public DynamicForm createNestedForm(Field f, String lang) {
       if (f.getRequired() != null && f.getRequired()) {
            throw new UnsupportedOperationException("inapplicable modifier for nested form");
        }
```

Reply:
@SykoraLukas @jkremlacek Co myslíte :-)?

Reply:
Může být, možná by stálo za to to dát do javadocu funkce, aby to v budoucnu nedělalo neplechu a dalo se to snáž najít.